app/src/main/cpp/Pointer.cpp

```diff
@@ -170,9 +170,14 @@ Pointer::SetTransform(const vrb::Matrix& aTransform) {
 }
 
 void
-Pointer::SetScale(const vrb::Vector& aHitPoint, const vrb::Matrix& aHeadTransform) {
-  const float scale = (aHitPoint - aHeadTransform.MultiplyPosition(vrb::Vector(0.0f, 0.0f, 0.0f))).Magnitude();
-  m.pointerScale->SetTransform(vrb::Matrix::Identity().ScaleInPlace(vrb::Vector(scale, scale, scale)));
+Pointer::SetScale(const float scale) {
+  if (m.layer) {
+    float size = kOuterRadius *  2.0f * scale;
+    m.layer->SetWorldSize(size, size);
+  } else {
+    m.pointerScale->SetTransform(vrb::Matrix::Identity().ScaleInPlace(
+              vrb::Vector(scale, scale, 1.0)));
```

Comment:
Edit: now I see that you need it for the next commit. Would it be possible to do that there? It looks like it's needed for that one but not strictly for this commit.

Reply:
> I guess the 1.0 it's because you don't care about scaling in Z right?

Right. If we scale on the Z axis the circles will be shown slightly raised above the widget behind. We want them to be drawn right on top, with no gap in-between.

> BTW I don't see a clear reason to change the signature of the method. I am not saying the old version is better, just that I prefer to minimize the amount of changes as much as possible (easier to review, easier to bisect, easier to revert, etc...).

I think the logic on how the scale is computed belongs outside the SetScale() method, in the caller. IMO SetScale() should only receive a scale value since Pointer class' doesn't need to know about its relation in the scene; that would be BrowserWorld's responsibility.

> Edit: now I see that you need it for the next commit. Would it be possible to do that there? It looks like it's needed for that one but not strictly for this commit.

Yes, I can move it into next commit; though I think it is clearer this way. Next commit is already difficult enough to follow as it is. Not a strong opinion though, will do as you prefer.